<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRule</class>
 <widget class="QWidget" name="FormRule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>预警类型 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxAlarm_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>协议类型 </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxProto_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddRule">
         <property name="text">
          <string>Add Rule</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDelRule">
         <property name="text">
          <string>Delete Rule</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableRule"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>预警类型</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxAlarm"/>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>协议类型</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxProto"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>源端地址 </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSrc"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>目的地址 </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditDst"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>包含内容 </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditContent"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>预警信息 </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditErrMsg"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="btnOK">
              <property name="text">
               <string>确  定</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancel">
              <property name="text">
               <string>取  消</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
